### DISCUSSION TITLE
docs: How to format code?

### DISCUSSION THREAD

**Discussion by parisholley** [mikro-orm/mikro-orm#3767] [General]
Is there a prettier process in the project so I can avoid unrelated diffs due to IDE formatting preferences?

**Comment by B4nan** [comment]
quite generic question. unrelated diffs of what? formatting of what? the ORM is code first, there are only few places where code gets generated.

we have this example for [formatting migrations](https://mikro-orm.io/docs/migrations#using-custom-migrationgenerator), i can imagine something similar to entity generator - although that is something you should do once when setting up new project with existing schema...

in general, if you want to use prettier, you set it up in your own project, and just use it. how is this related to using any ORM? even if something gets generated, you run prettier and it gets reformatted according to your setup.

**Comment by parisholley** [comment]
@B4nan you misunderstand, i am referring to contributions to this project, not when pulling it into my own
